frontera/contrib/backends/memory/__init__.py

```diff
@@ -128,11 +135,11 @@ def _get(self, obj):
 
     def update_cache(self, objs):
         objs = objs if type(objs) in [list, tuple] else [objs]
-        map(self._put, objs)
+        list(map(self._put, objs))
```

Comment:
I think it is because map returns a generator in PY3. So if we don't convert it to a list, it never get's applied.